robot/basestation/static/img/connection_icons/not_connected_bar.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Encoding=UTF-8
+Name=Link to wifi_bang
+Type=Link
+URL=https://2xom5j4aqh9o3quhfzkge6t1-wpengine.netdna-ssl.com/wp-content/uploads/2015/02/wifi_bang-300x300.png
+Icon=image-png
```

Comment:
same comment as above

Reply:
removed connectFolder with images, not being used